### PR TITLE
Restructuring Installation Section

### PR DESCRIPTION
**Pull request by tanneberger**
- pls take a look at #201 first 

## Things Done
- this reorders a couple of sections: 
   1.) Lingua-Franca Compiler (LFC) 
   2.) Lingo 
   3.) VScode 
   4.) Epoch
   
   I changed the ordering because most of the people are interested in installing the compiler, so this should be definitely the first section.

### CODE REVIEW

**Comment by lhstrh** [comment]
I've looked at this, but I'm not sure I agree that most users would first look for the CLI tools. My guess is that most people are interested in the VS Code extension... I've also requested a review from @cmnrd since he most recently worked on these docs.

File: docs/installation.md [hunk 1]
```diff
@@ -20,6 +20,69 @@ You can also spin up one of our pre-configured Cloud-based dev environments:
...
+
+If you'd like to contribute to Lingua Franca, you can find details about the recommended developer setup here.
+
+## Lingo the Lingua-Franca Package-Manager
+
+Lingo is the package-manager and build tool of the lingua-franca ecosystem.
+
+```
+    cargo install lingua-franca
+    PATH = $PATH:~/.cargo/bin
```
Comment (cmnrd) [inline docs/installation.md:83]: Shouldn't this be an export command?

File: docs/installation.md [hunk 2]
```diff
@@ -20,6 +20,69 @@ You can also spin up one of our pre-configured Cloud-based dev environments:
...
+
+Regular and nightly release builds of the command line tools can be downloaded from the [Lingua Franca release page](https://github.com/lf-lang/lingua-franca/releases). Download the archive that matches your OS and architecture, and extract the contents.
+
+### From Source
+
+Please refer to the [Lingua Franca GitHub repository](https://github.com/lf-lang/lingua-franca) for build instructions.
+
+If you'd like to contribute to Lingua Franca, you can find details about the recommended developer setup here.
+
+## Lingo the Lingua-Franca Package-Manager
```
Comment (cmnrd) [inline docs/installation.md:77]:
```suggestion
## Lingo
```

**Review by cmnrd — COMMENTED** [review]
I also think that we shouldn't change the order. Could you concert it back to the old order?

File: docs/installation.md (hunk 1 above)
Comment (tanneberger) [reply docs/installation.md:83]: the export is not necessary ;D it also works without it

**Comment by tanneberger** [comment]
Ok will revert to old order.

File: docs/installation.md (hunk 1 above)
Comment (cmnrd) [reply docs/installation.md:83]: Ok, right. It doen't work with the spaces around the `=` though.
```suggestion
    PATH=$PATH:~/.cargo/bin
```

**Comment by tanneberger** [comment]
changing the order

**Comment by cmnrd** [comment]
Didn't this also add install instructions for lingo? Those are currently missing on the website.

**Pull request closed without merging**